models/user/user.go

```diff
@@ -312,14 +311,25 @@ func (u *User) GenerateEmailActivateCode(email string) string {
 	return code
 }
 
-// GetUserFollowers returns range of user's followers.
-func GetUserFollowers(ctx context.Context, u, viewer *User, listOptions db.ListOptions) ([]*User, int64, error) {
-	sess := db.GetEngine(ctx).
-		Select("`user`.*").
+func searchUserFollowers(ctx context.Context, u, viewer *User) *xorm.Session {
+	return db.GetEngine(ctx).
 		Join("LEFT", "follow", "`user`.id=follow.user_id").
 		Where("follow.follow_id=?", u.ID).
 		And("`user`.type=?", UserTypeIndividual).
 		And(isUserVisibleToViewerCond(viewer))
+}
+
+func searchUserFollowing(ctx context.Context, u, viewer *User) *xorm.Session {
+	return db.GetEngine(ctx).
+		Join("LEFT", "follow", "`user`.id=follow.follow_id").
+		Where("follow.user_id=?", u.ID).
+		And("`user`.type IN (?, ?)", UserTypeIndividual, UserTypeOrganization).
+		And(isUserVisibleToViewerCond(viewer))
+}
+
+// GetUserFollowers returns range of user's followers.
+func GetUserFollowers(ctx context.Context, u, viewer *User, listOptions db.ListOptions) ([]*User, int64, error) {
+	sess := searchUserFollowers(ctx, u, viewer)
```

Comment:
Since `Select("`user`.*").` removed from `searchUserFollowers`, we need it here to reduce queries from joined tables.